js/blog.js

```diff
@@ -1,7 +1,288 @@
+/* Database configuration */
```

Comment:
this file is too long, break it down according to the features it holds
- have maybe other 2 files, just to have small clean manageable files